apps/rpc/src/modules/event/attendance-service.ts

```diff
@@ -282,7 +282,6 @@ export function getAttendanceService(
     async registerAttendee(handle, attendanceId, userId, options) {
       const attendance = await this.getAttendanceById(handle, attendanceId)
       const user = await userService.getById(handle, userId)
-      const event = await eventService.getByAttendance(handle, attendance.id)
```

Comment:
There is a ton of broken tests due to a regression from default find rules @brage-andreas. Adding this line again and running tests breaks 12 test cases.